code/client/src/core/ui/player_debug.cpp

```diff
@@ -0,0 +1,165 @@
+        ImGui::Text("IsThrownWeapon: %s\n", pActivePlayer->GetHumanWeaponController()->IsThrownWeapon() ? "true" : "false");
+
+        ImGui::Text("IsDeath: %s\n", pActivePlayer->IsDeath() ? "true" : "false");
+
+        ImGui::Text("IsInVehicle: %s\n", pActivePlayer->IsInVehicle() ? "true" : "false");
+        ImGui::Text("IsInCar: %s\n", pActivePlayer->IsInCar() ? "true" : "false");
+        ImGui::Text("IsInBoat: %s\n", pActivePlayer->IsInBoat() ? "true" : "false");
+
+        ImGui::Text("IsCrouch: %s\n", pActivePlayer->IsCrouch() ? "true" : "false");
+
+        ImGui::Text("IsAiming: %s\n", pActivePlayer->GetHumanWeaponController()->IsAiming() ? "true" : "false");
+
+        ImGui::Text("IsInCover: %s\n", pActivePlayer->IsInCover() ? "true" : "false");
+        ImGui::Text("IsInCoverFull(true): %s\n", pActivePlayer->IsInCoverFull(true) ? "true" : "false");
+        ImGui::Text("IsInCoverFull(false): %s\n", pActivePlayer->IsInCoverFull(false) ? "true" : "false");
+
+        bool lockedControls = Game::Helpers::Controls::AreControlsLocked();
+        if (ImGui::Checkbox("Lock Controls", &lockedControls)) {
+            Game::Helpers::Controls::Lock(lockedControls);
+        }
+
+        bool demigod = pActivePlayer->GetHumanScript()->IsDemigod();
+        if (ImGui::Checkbox("Demigod", &demigod)) {
+            pActivePlayer->GetHumanScript()->SetDemigod(demigod);
+        }
+
+        bool invulnerable = pActivePlayer->GetHumanScript()->GetInvulnerabilityByScript();
+        if (ImGui::Checkbox("Invulnerable", &invulnerable)) {
+            pActivePlayer->GetHumanScript()->SetInvulnerabilityByScript(invulnerable);
+        }
+
+        float health = pActivePlayer->GetHumanScript()->GetHealth();
+        if (ImGui::SliderFloat("Health", &health, 0.0f, pActivePlayer->GetHumanScript()->GetHealthMax())) {
+            pActivePlayer->GetHumanScript()->SetHealth(health);
+            (health);
+        }
+
+        /**
+         * SetHealthMax is explicitly disabled for the player
+         */
+        // float healthMax = pActivePlayer->GetHumanScript()->GetHealthMax();
+        // if (ImGui::SliderFloat("Health Max", &healthMax, 0.0f, 1000.0f)) {
+        //     pActivePlayer->GetHumanScript()->SetHealthMax(healthMax);
+        // }
+        ImGui::Text("Health Max: %f\n", pActivePlayer->GetHumanScript()->GetHealthMax());
+
+        {
+            if (ImGui::Button("Heal")) {
+                float healthMax = pActivePlayer->GetHumanScript()->GetHealthMax();
+                pActivePlayer->GetHumanScript()->SetHealth(healthMax);
+            }
+
+            ImGui::SameLine();
+
+            if (ImGui::Button("Heal to -1.0f")) {
+                pActivePlayer->GetHumanScript()->SetHealth(-1.0f); // Doesn't kill the player
+            }
+        }
+
+        /**
+         * TODO: How to kill the player?
+         */
+        // if (ImGui::Button("Kill")) {
+        //     // ??
+        // }
+
```

Comment:
And here. You can kill the player by setting its health to 0

Reply:
No, it doesn't work :/

See: 

https://github.com/MafiaHub/MafiaMP/blob/f1b4db0aeae2aad6c49b56f63fc9e417db7d107f/code/client/src/core/ui/player_debug.cpp#L117C1-L119C1

Reply:
Code removed since we have a dedicated issue for: https://github.com/MafiaHub/MafiaMP/issues/52